### ISSUE TITLE
conda out of date

### ISSUE THREAD

**Issue by Eh2406** [UDST/pandana#59]
attempting to run `conda update pandana` gives me 

```
# All requested packages already installed.
# packages in environment at C:\Anaconda2:
#
pandana                   0.1.2                    py27_0    synthicity
```
Looks like 0.2.0 was never loaded on conda for windows.

**Comment by sablanchard** [comment]
Yes that is correct this is on our list to update. For now pip installing should get you 0.2.0 correctly.

**Comment by stefancoe** [comment]
The API documentation lists the current version as 0.3.0. Has this been released? I'd like to be able to use the include_poi_id on the nearest_poi method, which is not available on 0.2.0. Thanks!

**Comment by pksohn** [comment]
Hi Stefan, `0.3.0` is available on [PyPI](https://pypi.python.org/pypi/pandana) and [conda via the UDST channel](https://anaconda.org/udst/pandana). Please let us know if you have issues installing it.

**Comment by stefancoe** [comment]
Thanks! Yeah- I was using the Synthicity channel because it is still linked in the documentation:
http://udst.github.io/pandana/installation.html

-Stefan

**Comment by pksohn** [comment]
Installation instructions are updated in docs: http://udst.github.io/pandana/installation.html. Now that pandana 0.3 is on conda and instructions are updated, I think this issue is all set.

**Comment by Holisticnature** [comment]
The udst channel provided by the documentation points to 3.0. Is there an ETA on 4.0 to the conda repo? I ran into a "reserve_num_graphs" issue that seems to be resolved in 4.0.

**Comment by sablanchard** [comment]
Hi @Holisticnature! Yes that is correct only 0.3 is available at the moment on conda I will reopen this issue to prompt the update to the conda builds. In the meantime you can install version 0.4 via pip.

**Comment by sablanchard** [comment]
@Holisticnature conda builds for pandana were updated last week to the latest release 0.4.1. Closing this issue. Please use `conda install -c udst pandana=0.4.1`
